app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -215,3 +219,11 @@ object IndividualDetails extends Randomiser {
   )
 
 }
+
+case class OrganisationDetails(name: String, address: Address)
+
+object OrganisationDetails extends Randomiser {
+
+  def random(): OrganisationDetails = OrganisationDetails(s"Company ${Random.nextInt(99999)}", Address.random())
```

Comment:
I think it is better as an alphanumeric https://github.com/hmrc/individual-income-des-stub/blob/b6b4b996d05c93bae00699a483f31a7593bbd3c0/app/uk/gov/hmrc/individualincomedesstub/service/Randomiser.scala#L42